content/docs/concepts/stacks.md

```diff
@@ -13,86 +13,79 @@ In the Getting Started tutorial, you used the Paketo builder to build a Node.js
 ## What is a stack?
 A `stack` consists of two images:
 * `build image`: the environment in which your app is built
-* `run image`: the OS layer for your app image
+* `run image`: the OS layer on which your app runs
 
-For more information about `stacks`, see [buildpacks.io](https://buildpacks.io/docs/concepts/components/stack/).
+To learn more about the concept of`stacks`, see
+[buildpacks.io](https://buildpacks.io/docs/concepts/components/stack/).
 
 ## What Paketo stacks are available?
-The Paketo project releases several stacks. These are:
-
-### Tiny
-
-#### Build Image
-{{< code/copyable >}}
-index.docker.io/paketobuildpacks/build:tiny-cnb
-{{< /code/copyable >}}
-
-#### Run Images
-{{< code/copyable >}}
-index.docker.io/paketobuildpacks/run:tiny-cnb
-{{< /code/copyable >}}
-{{< code/copyable >}}
-gcr.io/paketo-buildpacks/run:tiny-cnb
-{{< /code/copyable >}}
-
-#### Contains:
-* Build: ubuntu:bionic + openssl + CA certs + compilers + shell utilities
-* Run: distroless-like bionic + glibc + openssl + CA certs
-
-### Base
-#### Build Image
-{{< code/copyable >}}
-index.docker.io/paketobuildpacks/build:base-cnb
-{{< /code/copyable >}}
-
-#### Run Images
-{{< code/copyable >}}
-index.docker.io/paketobuildpacks/run:base-cnb
-{{< /code/copyable >}}
-{{< code/copyable >}}
-gcr.io/paketo-buildpacks/run:base-cnb
-{{< /code/copyable >}}
-
-#### Contains:
-* Build: ubuntu:bionic + openssl + CA certs + compilers + shell utilities
-* Run: ubuntu:bionic + openssl + CA certs
-
-### Full
-#### Build Image
-{{< code/copyable >}}
-index.docker.io/paketobuildpacks/build:full-cnb
-{{< /code/copyable >}}
-
-#### Run Images
-{{< code/copyable >}}
-index.docker.io/paketobuildpacks/run:full-cnb
-{{< /code/copyable >}}
-{{< code/copyable >}}
-gcr.io/paketo-buildpacks/run:full-cnb
-{{< /code/copyable >}}
-
-#### Contains:
-* Build: ubuntu:bionic + many common C libraries and utilities
-* Run: ubuntu:bionic + many common libraries and utilities
+The Paketo project releases several stacks. We currently support two Ubuntu
+distributions: 18.04 Bionic Beaver and 22.04 Jammy Jellyfish. Tiny, Base, and
+Full stack variants differ in the number of packages installed in the OS layer.
+The available stacks, _**from smallest to largest**_, are:
+- [Bionic Tiny](https://github.com/paketo-buildpacks/bionic-tiny-stack):
+  - Build image based on Ubuntu 18.04 Bionic Beaver; run image comparable to [distroless](https://github.com/GoogleContainerTools/distroless)
```

Comment:
@fg-j Looks like `distroless` is causing the spellcheck to fail. Isn't there a tag we can use hear to skip spellcheck?

Reply:
I've added this word to the dictionary, since it's a "real noun" in this context.